app/src/main/java/net/skyscanner/backpack/demo/MainActivity.kt

```diff
@@ -41,21 +64,90 @@ class MainActivity : BpkBaseActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_component_list)
```

Comment:
I think the layout file can be deleted now

Reply:
You mean delete the whole file? I tried deleting it but after that the app didn't run anymore and errors popped up. I've rolled back now and the app runs again.

Reply:
Yes the `activity_component_list` file. What's the error you get after deleting?

Reply:
I think I was trying to delete some other file. How do I find this one? Tried searching in the project but it doesn't come up.

Reply:
I usually use double shift (as in shift twice in a row) to search - ping me if you can't find it that way :)

Reply:
Found it now and deleted, thank you!